content/blog/2023/03/03/2023-03-09-jira-donation.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Jira for the Jenkins project"
+tags:
+- issues
+- jira
+authors:
+- markewaite
+opengraph:
+image: /images/post-images/2023/03/09/2023-03-09-jira-for-jenkins/jira-for-jenkins.png
+discourse: true
+---
+
+= Jira continues for Jenkins development
+
+The Jenkins project has used link:https://www.atlassian.com/software/jira[Jira Software] for issue tracking since 2005.
+Jira has been our issue and enhancement tracking system for almost as long as Jenkins core has existed.
+Jenkins core and most Jenkins plugins track issues and enhancements through a link:https://issues.jenkins.io[Jira] instance hosted for the Jenkins project by the Linux Foundation.
+
+image::/images/post-images/2023/03/09/2023-03-09-jira-for-jenkins/jira-for-jenkins.png[Atlassian sponsors Jenkins]
+
+We're pleased to announce that Atlassian has agreed to continue sponsoring the Jenkins project with a license to use Jira.
+We're deeply grateful to Atlassian for their support of open source software and especially for their support of the Jenkins project.
+We extend our thanks to the Linux Foundation as well for their continued hosting of Jira for the Jenkins project.
+
+== Upgrade schedule
+
+As part of the sponsorship, the Linux Foundation will need to update Jira and restart it.
```

Comment:
Do we know to which version? Then it might sense to write ... to update Jira to version 9.6 ...

Reply:
It is a license upgrade rather than a version upgrade.